test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/server/ui/editor/CRCPart.java

```diff
@@ -0,0 +1,44 @@
+/******************************************************************************* 
+ * Copyright (c) 2019 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.jboss.tools.cdk.reddeer.server.ui.editor;
+
+import org.eclipse.reddeer.core.matcher.WithTextMatcher;
+import org.eclipse.reddeer.swt.api.Button;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.text.LabeledText;
+import org.eclipse.reddeer.uiforms.impl.section.DefaultSection;
+import org.jboss.tools.cdk.reddeer.core.label.CDKLabel;
+
+/**
+ * 
+ * @author odockal
+ *
+ */
+public interface CRCPart {
+	
+	public DefaultSection getCDKDefaultSection();
+	
+	public default LabeledText getCRCBinary() {
+		return new LabeledText(getCDKDefaultSection(), "crc binary file: ");
```

Comment:
I would use for label also CDKLabel.java class, to have consistency around classes.